src/utils/sendEmail.js

```diff
@@ -0,0 +1,21 @@
+import sgMail from "@sendgrid/mail";
+
+const sendEmail = (email, verificationToken) => {
+  sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+  const msg = {
+    to: `${email}`,
+    from: "eliszewskipawel@gmail.com",
+    subject: "Verification link",
+    text: `To verify your account please go to http://localhost:4000/api/users/verify/${verificationToken}`,
```

Comment:
`http://localhost:4000` to jako wartość w .env. Potem domene kupisz/zmienisz to raczej lipa edytować kod.